hazelcast/src/main/java/com/hazelcast/internal/management/ManagementCenterService.java

```diff
@@ -125,21 +122,16 @@ public ManagementCenterService(HazelcastInstanceImpl instance) {
         this.timedMemberStateFactory = new TimedMemberStateFactory(instance);
         this.identifier = newManagementCenterIdentifier();
 
-        registerListeners();
+        if (this.managementCenterConfig.isEnabled()) {
```

Comment:
'this.' isn't needed. Only when it is ambiguous, e.g. a setter/constructor, this is recommended.